### DISCUSSION TITLE
¿Are there any size restrictions on notebooks uploaded to the repositories?

### DISCUSSION THREAD

**Discussion by akitxu** [community/community#134056] [Repositories] [answered]
### Select Topic Area

General

### Body

I'm trying to upload a Jupyter notebook to a repository and I find that it only uploads a part of it. The notebook only has 15.5MB. My notebook is 15.5MB in size. I have tried dragging the file to the repository and also updating the repository via http, and the same thing always happens.
Of the 16 notebook sections, I only see 12 in the repository. How can I fix this problem?
 Thanks for the help.

**Comment by tariqodeh0** [comment, marked as answer]
GitHub has a file size limit of 100 MB for individual files, and files over 50 MB may cause performance issues. To manage large Jupyter notebooks, you can split them into smaller parts, use Git Large File Storage (LFS) to handle large files efficiently, compress the notebook by removing unnecessary outputs, or optimize it by saving large datasets externally. Additionally, ensure your repository stays under 1 GB to avoid performance warnings.

**Reply by akitxu** [reply]
Thanks. It's strange, the notebook only has 15.5MB

**Reply by elsycharles** [reply]
Is there any ways to extends this size limit ?

**Reply by akitxu** [reply]
My notebook is 15.5MB in size. I have tried dragging the file to the repository and also updating the repository via http, and the same thing always happens.
Of the 16 notebook sections, I only see 12 in the repository. How can I solve this problem?

**Comment by akitxu** [comment]
The notebook with the problem is located at "https://github.com/akitxu/Python-Trading-Gestion-Conservadora/blob/master/Teoria/Estrategia_cruce_EMAs_3.ipynb". At the end of it you can see that the last output is item 34 and, below, there is one more line that seems to indicate that it has more items, which are not shown, up to a total of 40.
